README.md

```diff
@@ -1,8 +1,20 @@
 # multihash
 
+> hashpipe - pipe iff the hash matches
+
 Multihash is a protocol for differentiating outputs from various well-established cryptographic hash functions, addressing size + encoding considerations.
 
-It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See https://github.com/jbenet/random-ideas/issues/1 for a longer discussion.
+It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See [this GitHub thread](https://github.com/jbenet/random-ideas/issues/1) for a longer discussion.
+
+## Table of Contents
+
+- [Example](#example)
+- [Format](#format)
+- [Implementations:](#implementations)
+- [Table for Multihash v1.0.0-RC (semver)](#table-for-multihash-v100-rc-semver)
+- [Disclaimers](#disclaimers)
+- [Contribute](#contribute)
+- [License](#license)
```

Comment:
Its part of the standard. I think it's important, because it saves a lot of time just glancing at the document, and you don't know the length of their output screen. Could be using less or something, with only 30 lines.